### ISSUE TITLE
[Bug] Jerky flashing scrolling title text in tabs tray

### ISSUE THREAD

**Issue by PurpleCodingWizard** [mozilla-mobile/fenix#15216]
## Steps to reproduce
1. load website https://www.cybercomplaint.in/complaintcate?id=MjY2NQ==/Other/Mobile-lost-21-8-2018-22:56
2. Jump to tabs tray view
3. Watch behavior in action: https://imgur.com/a/SbAMFcB 
### Expected behavior
1. Tab title does not scroll remain static
2. Else make it easier on the eyes with optional toggle to disable in settings.
### Actual behavior
Title scrolls by flashing this tab's view when JavaScript is active. 
Blocking JavaScript with ublock-origin stops scroll.
### Device information

* Android device: 10
* Fenix version: Nightly 200918 15:19 (Build #2015764665)

**Comment by s-ankur** [comment]
I think they are using the title marquee trick.

**Comment by ekager** [comment]
Yes this seems like the site intentionally changing their title

**Comment by PurpleCodingWizard** [comment]
I still think it should be controllable by the user but alright. @ekager 

 I failed to mention it clearly in text description. But, back then the reason I filed this bug was because Firefox was flashing/flickering the UI of this tab's list view heavily. It happens when scrolling the title text in it. Can something not be done to remove the UI flash?
It can still be reproduced in latest Firefox 84 on both list or grid settings view. It will flash/flicker the screen like older versions worse in list view. This is annoying and a pain for the eyes. I also see it as an accessibility issue which should be fixed later.
